packages/fxa-auth-db-mysql/db-server/test/backend/db_tests.js

```diff
@@ -2538,101 +2538,134 @@ module.exports = function(config, DB) {
     });
 
     describe('db.securityEventsByUid', () => {
-      let session1, session2, session3, anotherAccountSession, uid, anotherUid
-      const evA = 'account.login', evB = 'account.create', evC = 'account.reset'
-      const addr = '127.0.0.1'
+      let session1, session2, session3, anotherAccountSession, uid, anotherUid;
+      const evA = 'account.login',
+        evB = 'account.create',
+        evC = 'account.reset';
+      const addr = '127.0.0.1';
```

Comment:
Prettier is complicating the diff here. The substantive change is further down...